<launch>
<!-- Lança o nó Physarum com parâmetros -->
  <node name="physarum_node_robot_1"
        pkg="physarum_simulation"
        type="physarum_node.py"
        output="screen">
    <param name="robot_name" value="robot_0"/>
    <param name="agent_count" value="2000"/>
    <param name="width" value="100"/>
    <param name="height" value="100"/>
  </node>
  
  <node name="physarum_node_robot_2"
        pkg="physarum_simulation"
        type="physarum_node.py"
        output="screen">
    <param name="robot_name" value="robot_1"/>
    <param name="agent_count" value="2000"/>
    <param name="width" value="100"/>
    <param name="height" value="100"/>
  </node>

  <node name="physarum_node_robot_3"
        pkg="physarum_simulation"
        type="physarum_node.py"
        output="screen">
    <param name="robot_name" value="robot_2"/>
    <param name="agent_count" value="2000"/>
    <param name="width" value="100"/>
    <param name="height" value="100"/>
  </node>
  <!-- <node name="controle"
        pkg="physarum_simulation"
        type="coletar_conectividade_global.py"
        output="screen">
        <param name="output_path" value="$(env HOME)/physarum_dados/sim_001.csv"/>
  </node> -->
    <!-- <node name="controle2"
        pkg="physarum_simulation"
        type="robustez.py"
        output="screen">
        <param name="output_path" value="$(env HOME)/physarum_dados/sim_001.csv"/>
  </node> -->
</launch>
